packages/dds/task-manager/package.json

```diff
@@ -23,6 +23,16 @@
 				"default": "./dist/index.js"
 			}
 		},
+		"./legacy": {
+			"import": {
+				"types": "./lib/legacy.d.ts",
+				"default": "./lib/index.js"
+			},
+			"require": {
+				"types": "./dist/legacy.d.ts",
+				"default": "./dist/index.js"
+			}
+		},
```

Comment:
'Task-Manager' previously had no `@alpha` tags so the '/legacy' export was missing.